core-designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/icon/NiaIcons.kt

```diff
@@ -46,7 +46,7 @@ import com.google.samples.apps.nowinandroid.core.designsystem.R
 object NiaIcons {
     val AccountCircle = Icons.Outlined.AccountCircle
     val Add = Icons.Rounded.Add
-    val ArrowBack = Icons.Rounded.ArrowBack
+    val ArrowBack = Icons.Filled.ArrowBack
```

Comment:
The design system specifically calls for the rounded icon. Please revert this.

Reply:
Done